### PR TITLE
ADD per service notif queues

### PR DESCRIPTION
**Pull request by fgalan**
Issue #3843

- [x] CLI (and cli.md)
- [x] ftest
- [x] Review pending "FIXME PR"
- [x] New section in perf tunning doc
- [x] CNR
- [x] devel manual
- [x] weird boost error message upon Ctrl+C signal - solved in c56ecdc

### CODE REVIEW

**Comment by fgalan** [comment]
When Ctrl+C the contextBroker process with threadpool in use (either with or without specific service pools) the following error appears:

```
contextBroker: /usr/include/boost/thread/pthread/condition_variable_fwd.hpp:102: boost::condition_variable::~condition_variable(): Assertion `!ret' failed.
Abortado
```

File: doc/manuals/devel/sourceCode.md [hunk 1]
```diff
@@ -448,10 +448,6 @@ When an entity is created or modified or when a subscription is created or modif
 [**mongoBackend**](#srclibmongobackend) decides when to notify and **ngsiNotify** executes the notification, with the help of the external library [libcurl](https://curl.haxx.se/libcurl/).
 Actually, a function from the [**rest** library](#srclibrest) is used: `httpRequestSend()`. Another important aspect of this library is that the notifications are sent by separate threads, using a thread pool if desired.
 
-We have subscriptions both for *context entities* and for *availability registrations* and thus, there are two types of notifications. Context entity subscriptions are by far more important (registration subscriptions are barely used at all) and the thread pool for notifications is used for context entity subscriptions only.
-
-### Context entities notifications
-
```
Comment (fgalan) [inline doc/manuals/devel/sourceCode.md:454]: An old leftover of context availability subscription which no longer makes sense in this documentation. I have taken the opportunity to clean it up.

NTC (informative)

**Review by mrutid — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR modifies some .md files. It would be great if you could do a PR to sync the Japanese translation, please. Thanks!

**Comment by fisuda** [comment]
I have sent the PR #3870. It would be great if you could review it.

**Pull request merged**